lib/mapUtils.js

```diff
@@ -23,7 +23,8 @@ function createProductData(record) {
     id: data.id,
     category: data.Category,
     points: data.Points,
-    customerCost: data['Customer Cost']
+    customerCost: data['Customer Cost'],
+    image: data.Image ? data.Image[0].url : null
```

Comment:
This is much cleaner than the full if/else! We should update Clerk to do this too @thumn

Reply:
sounds good!